src/mlpack/core/optimizers/cne/cne.hpp

```diff
@@ -0,0 +1,231 @@
+/**
+ * @file cne.hpp
+ * @author Marcus Edel
+ * @author Kartik Nighania
+ *
+ * Conventional Neural Evolution
```

Comment:
Can you elaborate on this, a simple sentence what this method does is just fine.